contracts/near-htlc/scripts/deploy_testnet.sh

```
@@ -0,0 +1,221 @@
+
+# Function to create test escrow
+create_test_escrow() {
+    local contract_id="$1"
+    local resolver_id="$2"
+    local beneficiary_id="$3"
+    
+    echo -e "${YELLOW}Creating test escrow...${NC}"
+    
+    local secret="test_secret_12345"
+    local secret_hash=$(echo -n "$secret" | sha256sum | cut -d' ' -f1)
+    local secret_hash_base58=$(echo -n "$secret_hash" | xxd -r -p | base58)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)